templates/endpoints_v2/cookbooks/vision_language_model/vision_language_model/README.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy, configure, and serve LLMs \n",
+    "\n",
+    "**⏱️ Time to complete**: 10 min\n",
+    "\n",
+    "Ready for some adventure into multi-modal models? Vision language model support is now added to RayLLM. The current support includes LlaVA-NeXT models with dynamic image resolution. Meta AI’s Chameleon, with its early fusion architecture, will come soon. \n",
+    "\n",
+    "In this example, we will use RayLLM to serve [llava-hf/llava-v1.6-mistral-7b-hf](https://huggingface.co/llava-hf/llava-v1.6-mistral-7b-hf). We will build a Gradio application that brings art to life with ekphrasis, specially designed for kids. Imagine turning kid drawings into whimsical verses and delightful rhymes. Now let’s buckle up and dive into the fun! 🎨🚀\n",
```

Comment:
Change the tone to sth more doc like :). This is not meant to be for twitter. Is it ChatGPT or you? :D If chat gpt prompt it to not be so cheesy (JK) :D

Reply:
But you should choose your phrases in a way that it will remain reliable with time. For example `chemelon coming soon`. When chamelon is supported we have to update templates. That's extra work. Make this doc generic and write it in a way that remains invariant with time.